Sample-test/sample.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!-- <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
```

Comment:
If this is commented this can also be removed

Reply:
yes